package.json

```diff
@@ -53,6 +55,7 @@
     "@types/react-dom": "^17.0.0",
     "@whitespace/storybook-addon-html": "^5.0.0",
     "clsx": "^1.1.1",
+    "concurrently": "^7.2.1",
```

Comment:
Why is concurrently added? Where is it being used?

Reply:
It is used i the scripts in package.json https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/main/package.json#L11

Reply:
I just can't comprehend how concurrently was not a part of package.json in the first place.
As far as I know me and Adam has not been installing it globally on our workstations.
It's a bit of a mystery

Reply:
It was missing...